src/io.jl

```diff
@@ -5,7 +5,7 @@ export create_internal_tables!, export_solution_to_csv_files
 
 Creates internal tables.
 """
-function create_internal_tables!(connection)
+function create_internal_tables!(connection, skip_validation = false)
```

Comment:
Should it be a keyword? I mean something like this:

```suggestion
function create_internal_tables!(connection; skip_validation = false)
```

Reply:
Yes, good catch